lib/matplotlib/path.py

```diff
@@ -330,6 +330,7 @@ def make_compound_path(cls, *args):
         if not args:
             return Path(np.empty([0, 2], dtype=np.float32))
 
+        # concatenate paths
```

Comment:
not sure the comment adds much?

Reply:
👍 

I tend to leave these "section" comments on functions that require blank lines for readability. 

Guess it's a moot point though since if we're not including the extra code to respect a terminal "STOP", then the function is now so short that this comment becomes *completely* superfluous.